src/store.js

```diff
@@ -71,6 +73,7 @@ export class Store {
         );
 
         const persistentState = {
+            cacheVersion: CACHE_VERSION,
```

Comment:
Any arguments against calling it just `version` instead of cacheVersion?

Reply:
No arguments. Let's do it.